subsys/nfc/lib/platform.c

```diff
@@ -190,3 +322,57 @@ void nfc_platform_event_handler(nrfx_nfct_evt_t const *event)
 		break;
 	}
 }
+
+
+#ifdef CONFIG_NFC_THREAD_CALLBACK
+void nfc_platform_cb_request(void *ctx,
+			     size_t ctx_len,
+			     const uint8_t *data,
+			     size_t data_len,
+			     bool copy_data)
+{
+	uint32_t item_buf_tx[MAX_ITEM_SIZE32];
+	size_t item_len;
+	uint8_t item_len32;
+	int err;
+	uint8_t *buf = (uint8_t *)item_buf_tx;
+
+	item_len = ctx_len;
+
+	memcpy(buf, ctx, ctx_len);
+
+	if (copy_data) {
+		memcpy(&buf[ctx_len], data, data_len);
+		item_len += data_len;
+	} else {
+		*((uint32_t *)(&buf[ctx_len])) = POINTER_TO_UINT(data);
+		item_len += sizeof(uint32_t);
+	}
+
+	item_len32 = ROUND_UP(item_len, sizeof(uint32_t));
```

Comment:
Just `ceiling_fraction(item_lem, sizeof(uint32_t))`. It seems like the result will be the same like for those two lines here. Could you check ?